src/app/clients/edit-client/edit-client.component.ts

```diff
@@ -101,11 +102,12 @@ export class EditClientComponent implements OnInit {
       'externalId': [''],
       'genderId': [''],
       'mobileNo': [''],
+      'emailAddress': ['', Validators.email],
```

Comment:
Same as above for the validations, it's a plan text field, I don't think this is needed.